### ISSUE TITLE
Modal Content Size to Full Size

### ISSUE THREAD

**Issue by ArEnSc** [nonameplum/SwiftUIModal#3]
@nonameplum 
Hey I was wondering, I was working on something similar, is it possible to have it switch between the content size and full? I don't think so from content size -> full and back 
This is due to the limitation on the modal right? there isn't a hack to get this to work? thanks!

**Comment by nonameplum** [comment]
If you mean the presentation style then probably no.
But if you only need that in terms of the UI perspective you could try to do it ourself by fill-out the screen with the bottom sheet and then back. You might try to also check [`presentationDetents`](https://developer.apple.com/documentation/swiftui/view/presentationdetents(_:)).

**Comment by ArEnSc** [comment]
> If you mean the presentation style then probably no. But if you only need that in terms of the UI perspective you could try to do it ourself by fill-out the screen with the bottom sheet and then back. You might try to also check [`presentationDetents`](https://developer.apple.com/documentation/swiftui/view/presentationdetents(_:)).
yeah I was trying to get backwards compatibility to have half detents and large detents
